src/txdb.h

```diff
@@ -125,6 +125,19 @@ class CBlockTreeDB : public CDBWrapper
     bool LoadBlockIndexGuts(const Consensus::Params& consensusParams, std::function<CBlockIndex*(const uint256&)> insertBlockIndex);
 };
 
+class BaseIndexDB : public CDBWrapper
+{
+public:
+    BaseIndexDB(const fs::path& path, size_t n_cache_size,
+                bool f_memory = false, bool f_wipe = false, bool f_obfuscate = false);
+
+    /// Read block locator of the chain that the txindex is in sync with.
+    bool ReadBestBlock(CBlockLocator& locator) const;
```

Comment:
Remove `ReadBestBlock` and `WriteBestBlock` from `TxIndexDB` definition.

Reply:
Good catch, thanks.